e2e/testrunner.test.js

```diff
@@ -1,10 +1,28 @@
-import assert from 'assert'
+import fs from 'fs'
+import path from 'path'
 import Launcher from '../packages/wdio-cli/src/launcher.js'
 
-jest.setTimeout(1000 * 60)
-
 test('should allow to run multiple browser at once', async () => {
     const launcher = new Launcher(`${__dirname}/wdio/wdio.conf.js`)
     const failures = await launcher.run()
-    assert.equal(failures, 0)
+    const hasPassed = failures === 0
+
+    /**
+     * print log files for debugging if test fails
+     */
+    if (!hasPassed) {
+        const rootPath = path.join(__dirname, 'wdio')
+        const logFiles = fs.readdirSync(rootPath).filter((file) => (
+            // only log files
+            file.endsWith('.log')
```

Comment:
This filter does not work as `file.endsWith('.log')` needs to be returned here.

Reply:
it returns because the surrounding `(` and `)`

Reply:
made it more clear